### DISCUSSION TITLE
Music 'primes' Whisper to hear only music

### DISCUSSION THREAD

**Discussion by ArthurPeabody** [ggml-org/whisper.cpp#574] [General]
I've been transcribing a series of radio programs, all recorded with the same quality and voices.  I start them before I go to bed because they hog CPU time and take 4 hours.  I woke up this morning to find Whisper had transcribed last night's program as all '[Music]'.  I suspect the problem was that the first 50 seconds was music.  If I start it after that it works as usual.

**Comment by ofou** [comment]
Same problem here. Any workarounds? I realized that there's music in the background.

**Reply by ArthurPeabody** [reply]
My problem was that the first 50 seconds was only music.  I restarted at second 51 and had the usual result.  Whisper has usually done a good job of ignoring background music for me.  If the music is in a separate channel you can separate channels.
